tap_zendesk/streams.py

```diff
@@ -21,52 +25,309 @@
+            )),
+        )),
+        th.Property("metadata", th.ObjectType(
+            th.Property("custom", th.ObjectType(additional_properties=True)),
+            th.Property("trusted", th.BooleanType),
+            th.Property("notifications_suppressed_for", th.ArrayType(th.IntegerType)),
+            th.Property("flags_options", th.ObjectType(
+                th.Property("2", th.ObjectType(
+                    th.Property("trusted", th.BooleanType)
+                )),
+                th.Property("11", th.ObjectType(
+                    th.Property("trusted", th.BooleanType),
+                    th.Property("message", th.ObjectType(
+                        th.Property("user", th.StringType)
+                    ))
+                ))
+            )),
+            th.Property("flags", th.ArrayType(th.IntegerType)),
+            th.Property("system", th.ObjectType(
+                th.Property("location", th.StringType),
+                th.Property("longitude", th.NumberType),
+                th.Property("message_id", th.StringType),
+                th.Property("raw_email_identifier", th.StringType),
+                th.Property("ip_address", th.StringType),
+                th.Property("json_email_identifier", th.StringType),
+                th.Property("client", th.StringType),
+                th.Property("latitude", th.NumberType)
+            ))
+        )),
+        th.Property("attachments", th.ArrayType(th.ObjectType(
+            th.Property("id", th.IntegerType),
+            th.Property("size", th.IntegerType),
+            th.Property("url", th.StringType),
+            th.Property("inline", th.BooleanType),
+            th.Property("height", th.IntegerType),
+            th.Property("width", th.IntegerType),
+            th.Property("content_url", th.StringType),
+            th.Property("mapped_content_url", th.StringType),
+            th.Property("content_type", th.StringType),
+            th.Property("file_name", th.StringType),
+            th.Property("thumbnails", th.ArrayType(th.ObjectType(
+                th.Property("id", th.IntegerType),
+                th.Property("size", th.IntegerType),
+                th.Property("url", th.StringType),
+                th.Property("inline", th.BooleanType),
+                th.Property("height", th.IntegerType),
+                th.Property("width", th.IntegerType),
+                th.Property("content_url", th.StringType),
+                th.Property("mapped_content_url", th.StringType),
+                th.Property("content_type", th.StringType),
+                th.Property("file_name", th.StringType)
+            )))
+        )))
+    ).to_dict()
+
+    def get_url_params(
+            self,
+            context: dict | None,
+            next_page_token: Any | None,
+    ) -> dict[str, Any]:
+        """Return a dictionary of values to be used in URL parameterization."""
+        params = {"sort": "created_at"}
```

Comment:
Not sure if it's required to sort by `created_at`. We will collect from many tickets and it will not be sorted (if we consider all data).